protostar/cairo/cairo_function_runner_facade.py

```diff
@@ -59,3 +59,21 @@ def get_return_values(self, n_ret: int):
             self._previous_runner
         ), "No runs were performed, so no return values are available!"
         return self._previous_runner.get_return_values(n_ret)
+
+    def is_panic(self) -> bool:
+        result = self.get_return_values(3)
+        return result[0] != 0
+
+    def get_panic_data(self) -> list[int]:
+        assert self.is_panic()
```

Comment:
This way we're forced to use this pattern:
```
if is_panic():
  get_panic_data()
```

Can't we just do `get_panic_data` and if there was no panic, it will return an empty array? And then, `is_panic` could be just:
```
def is_panic(self) -> bool:
  return not self.get_panic_data()
```